# TF use
use_tf_scan_transformation: true
use_tf_pose_start_estimate: false
pub_map_odom_transform: false
pub_map_scanmatch_transform: false
pub_odometry: false # not used, TODO: check if useful or not

# Map size/ start point
map_resolution: 0.15
map_size: 1024 # ~164m
map_start_x: 0.5
map_start_y: 0.5
map_multi_res_levels: 3
map_with_known_poses: false

# Map update parameters
map_pub_period: 0.5
output_timing: true
update_factor_free: 0.4
update_factor_occupied: 0.9
map_update_distance_thresh: 0.0
map_update_angle_thresh: 0.0
laser_z_min_value: -1.5
laser_z_max_value: 1.5
laser_min_dist: 0.3
laser_max_dist: 15.0

# Advertise config
advertise_map_service: true
scan_subscriber_queue_size: 5

# Debug parameters
output_timing: true
pub_drawings: false
pub_debug_output: false


